src/eq_schema/Answer.js

```diff
@@ -35,6 +35,7 @@ class Answer {
     { id, properties, type, label, description },
     parentAnswerId
   ) {
+    assign(properties, { required: false });
```

Comment:
Rather than making the change here, I think a better option would be to change the API so that the mandatory property defaults to false for child answers https://github.com/ONSdigital/eq-author-api/blob/master/repositories/strategies/multipleChoiceOtherAnswerStrategy.js#L11